product_docs/docs/epas/15/epas_compat_bip_guide/03_built-in_packages/18_dbms_utility.mdx

```diff
@@ -384,6 +386,29 @@ return VARCHAR2
 
 You can use this function in a stored procedure, function, or package to return the current call stack in a readable format. This function is useful for debugging.
 
+## FORMAT_ERROR_BACKTRACE
+
+The `FORMAT_ERROR_BACKTRACE` function returns the current error call stack, i.e. function name and lines that lead up to the exception.
+
+```sql
+DBMS_UTILITY.FORMAT_ERROR_BACKTRACE
+return VARCHAR2
+```
+
+This function can be used in a stored procedure, function or package to return the current call backtrace in a readable format. This function is useful for debugging purposes.
```

Comment:
This function can be used in a stored procedure, function, or package to return the current call backtrace in a readable format. This function is useful for debugging purposes.